INSS.EIIR.DataSync.Infrastructure/Sink/XML/IirXMLWriterHelper.cs

```diff
@@ -48,7 +48,7 @@ public static void WriteIirReportRequestToStream(InsolventIndividualRegisterMode
 
                 }
 
-                if (model.IncludeCaseDetailsInXML)
+                if (model.IncludeCaseDetailsInXML(DateTime.Now))
```

Comment:
why we are not using DateTime.UtcNow  ?

Reply:
In this particular case the original tested used Datetime.Now, the stored proc from which it was derived used GETDATE.  There is no requirement to change it to UTC.  App Service is configured to use localtime.  Passing of Datetime has been parameterised like this to allow unit testing of logic.